recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  fn: mayavi-{{ version }}.tar.gz
+  url: https://github.com/enthought/mayavi/archive/{{ version }}.tar.gz
+  sha256: 36f688b3ea542e9f8cc0d7faa25e1425723cd00acc8aa640169029f33679ab85
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - numpy x.x
+    - setuptools
+    - vtk
```

Comment:
- python : 2.7 and 3.5 are supported
- vtk: I think yes, we shoud pin it to >= 7.1